modules/admin_manual/pages/configuration/server/occ_command.adoc

```diff
@@ -3319,7 +3319,33 @@ Example:
 {occ-command-example-prefix} notifications:generate -g Office "Emergency Alert" "Rebooting in 5min"
 ----
 
-== ownCloud Maintained Application Commands
+=== Migration Steps Command
+
+You can run migration steps with the `migrations` command.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} migrations:execute <app> <version>
+----
+
+===== Arguments
+
+[width="100%",cols="20%,70%",]
+|===
+| `app`  | Name of the app this migration command shall work on.
+| `version`  | The version to execute.
+|===
+
+===== Example
+
+This example executes the migration step for the core app:
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} migrations:execute core 20181220085457
+----
+
+== Apps Commands
```

Comment:
Is this header still necessary?

Reply:
why not?

Reply:
How's it different to https://github.com/owncloud/docs/pull/1651/files#diff-2818bba1033f9f6297d7e1437dd4370eR153?

Reply:
why not just write the words? It's just an "s".

Instead of this, you do https://en.wikipedia.org/wiki/Cut,_copy,_and_paste and make people https://en.wikipedia.org/wiki/Reading and https://en.wikipedia.org/wiki/Research

But to the matter at hand: App commands sounds right to me, because what follows are the commands ( many of them ) to an app ( one at the time )

Apps commands sounds chaotic at first, and then it would also assume that what follows are commands for multiple apps.

App commands sounds clean. That's why I changed it. If you feel otherwise please explain.